tests/fixtures/addon/with-concat-files/index.js

```diff
@@ -0,0 +1,19 @@
+/* jshint node: true */
+'use strict';
+
+var path = require('path');
+
+module.exports = {
+  name: 'concat-some-files',
+
+  included: function(app) {
+    this._super.included(app);
+
+    var vendor = this.treeGenerator(path.join(__dirname, 'vendor'));
+
+    return this.concatFiles(vendor, {
```

Comment:
I'm not sure why concatFiles was introduced honestly. I thought we learned our lesson after exposing this.pickFiles and such :smiley_cat: